components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -2737,7 +2737,7 @@ private boolean isMemberAttributeRequired(Map<String, Boolean> requiredAttribute
     }
 
     @Override
-    public void deleteGroup(String groupId) throws NotFoundException, CharonException {
+    public void deleteGroup(String groupId) throws NotFoundException, CharonException, BadRequestException {
```

Comment:
Won't this be a breaking change to code that relies on handling exceptions NotFoundException and CharonException